src/core/client/auth/components/EmailField.tsx

```diff
@@ -13,33 +13,49 @@ import { ValidationMessage } from "coral-ui/components/v3";
 
 interface Props {
   disabled: boolean;
+  focus?: boolean;
 }
 
-const EmailField: FunctionComponent<Props> = (props) => (
-  <Field name="email" validate={composeValidators(required, validateEmail)}>
-    {({ input, meta }) => (
-      <FormField>
-        <Localized id="general-emailAddressLabel">
-          <InputLabel htmlFor={input.name}>Email Address</InputLabel>
-        </Localized>
-        <Localized
-          id="general-emailAddressTextField"
-          attrs={{ placeholder: true }}
-        >
-          <TextField
-            {...input}
-            id={input.name}
-            placeholder="Email Address"
-            type="email"
-            color={streamColorFromMeta(meta)}
-            disabled={props.disabled}
-            fullWidth
-          />
-        </Localized>
-        <ValidationMessage meta={meta} />
-      </FormField>
-    )}
-  </Field>
-);
+const EmailField: FunctionComponent<Props> = (props) => {
+  let inputRef: HTMLInputElement | null = null;
```

Comment:
This should use `useRef` instead:

```suggestion
const inputRef = useRef<HTMLInputElement>(null);
```